src/error/__tests__/formatError-test.ts

```diff
@@ -8,8 +6,8 @@ import { GraphQLError } from '../GraphQLError';
 
 describe('formatError: default error formatter', () => {
   it('uses default message', () => {
-    // $DisableFlowOnNegativeTest
-    const e = new GraphQLError();
+    // @ts-expect-error
+    const e = new GraphQLError(undefined);
```

Comment:
Why you can't call it without `undefined`?

Reply:
Because `message` is not optional, and it must be `string`:

```
export class GraphQLError extends Error {
  constructor(
    message: string,
    nodes?: Maybe<ReadonlyArray<ASTNode> | ASTNode>,
    source?: Maybe<Source>,
    positions?: Maybe<ReadonlyArray<number>>,
    path?: Maybe<ReadonlyArray<string | number>>,
    originalError?: Maybe<Error>,
    extensions?: Maybe<{ [key: string]: any }>,
  );
```

Reply:
But you wrote `@ts-expect-error` one line above and it's the negative test?

Reply:
Ohh you are right, fixed :)